app/lib/frontend/templates/views/layout.mustache

```diff
@@ -31,9 +31,11 @@
   <meta name="description" content="{{& pageDescription}}" />
   <meta property="og:description" content="{{& pageDescription}}" />
   <link rel="alternate" type="application/atom+xml" title="Updated Packages Feed for Pub" href="/feed.atom" />
+  <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet" type="text/css" />
```

Comment:
While the unpkg.com seems to be okay, for versioning and also to reduce third-party risk, we should download these files and have them referenced locally.